### ISSUE TITLE
Locked accounts are not redirected to lockout page in dashboard

### ISSUE THREAD

**Issue by javitonino** [CartoDB/cartodb#14310]
### Steps to Reproduce
1. Lock an account (set state = locked)
2. Go to dashboard

### Current Result
Dashboard displays with errors (APIs are blocked by backend)

### Expected result
Redirection to /lockout

This was previously implemented in backend and was not migrated to the new dashboard statics. Implemented in https://github.com/CartoDB/cartodb/blob/master/app/controllers/application_controller.rb#L198-L207

Basically, `if (user.state == 'locked') { redirect_to '/lockout' }` or something along those lines. I think `state` might not be available in `/me` but we can add it.

@CartoDB/rt-managers

**Comment by Jesus89** [comment]
Done in https://github.com/CartoDB/cartodb/pull/14344

**Comment by Jesus89** [comment]
In production :rocket:
